docker-compose/cassandra-3.11/.env

```diff
@@ -1,4 +1,5 @@
 COMPOSE_PROJECT_NAME=cass311-stargate
-CASSTAG=3.11.13
+CASSTAG=3.11
```

Comment:
just to confirm `3.11` always points to latest? This is good, but could end up into situation where docker is started with the non-supported version. for example coordinator supports `3.11.13` and C* moves to `3.11.14`.. But maybe this is also fine, especially if we can have this in a CI that could run stuff on top of this?

Reply:
By definition a Cassandra `3.11.14` node should be backward compatible. We can revisit if this proves not to be the case. My thought was that we don't remember to update this file regularly when Cassandra patch version increases, so let's take that off the table.